fhir-swagger-generator/src/main/java/com/ibm/fhir/openapi/generator/FHIROpenApiGenerator.java

```diff
@@ -1286,8 +1288,14 @@ private static void generateProperty(StructureDefinition structureDefinition, Cl
             JsonObjectBuilder wrapper = factory.createObjectBuilder();
             wrapper.add("type", "array");
             wrapper.add("items", property);
+            if (example != null) {
+                wrapper.add("example", example);
+            }
             properties.add(elementName, wrapper);
         } else {
+            if (example != null) {
```

Comment:
Is this always false?

Reply:
no, I think github was just hiding the lines in the middle on you.  be sure to expand the 
<img width="101" alt="image" src="https://user-images.githubusercontent.com/5478798/82272940-0e041680-994a-11ea-8ed6-019d71d6d32f.png"> to see the full context